### PR TITLE
Create tag-security-questionnaire.md

### PR DESCRIPTION
**Pull request by johannhof**
As part of the Blink launch process we would like to get a TAG review for SAA, for which we need a S&P questionnaire (separately from the S&P considerations).

@bvandersloot-mozilla want to take a look? This is partly copied from your considerations :)

cc @mreichhoff @cfredric

### CODE REVIEW

File: tag-security-questionnaire.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+12.  What temporary identifiers do the features in this specification create or
+     expose to the web?
+     
+None
+
+13.  How does this specification distinguish between behavior in first-party and
+     third-party contexts?
+
+This specification is meant to be used in third-party contexts with a default grant behavior in first-party contexts.
```
[the remark this answers is not in the document]
Comment (annevk) [reply tag-security-questionnaire.md:76]: We should add a question here about https://tess.oconnor.cx/2020/10/parties to see if they are paying attention.

**Review by annevk — COMMENTED** [review]
This looks okay to me, modulo https://tess.oconnor.cx/2020/10/parties.

File: tag-security-questionnaire.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bvandersloot-mozilla) [reply tag-security-questionnaire.md:76]: Hah! I take first-party here to mean same-origin, which makes the answer make sense to me.

[the remark this answers is not in the document]
Comment (annevk) [reply tag-security-questionnaire.md:76]: Well, what I was saying is that I think it only works for cross-site contexts at the moment. I don't think our logic holds up if you applied it same-origin. (You can sorta pretend it's similar, but that's different.)

[the remark this answers is not in the document]
Comment (bvandersloot-mozilla) [reply tag-security-questionnaire.md:76]: Ah, I think I see what you mean. There are a few wrinkles for those non-same-origin cookies which are "first-party" and same-site cookies which are "third-party"

**Review by bvandersloot-mozilla — APPROVED** [review]
Looks like a faithful communication of the considerations section (and then some) :+1:

File: tag-security-questionnaire.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (johannhof) [reply tag-security-questionnaire.md:76]: @annevk apologies, I don't think I understand what you're saying/suggesting here. Is this about whether the spec will do same-origin or same-site checks when determining storage access? I feel like we should probably fix the spec to consider the cookie-level site boundary (i.e. run same-site checks vs. same origin).

[the remark this answers is not in the document]
Comment (annevk) [reply tag-security-questionnaire.md:76]: It depends on how you read this sentence, but I don't think "default grant behavior in first-party contexts" actually works. Rather, the permission is not consulted in first-party contexts.

End users or browsers cannot block "storage-access" in a first-party context in a meaningful way through this specification. It's only designed for third-party contexts.

[the remark this answers is not in the document]
Comment (johannhof) [reply tag-security-questionnaire.md:76]: Okay, yeah, I think that makes sense, I'll modify the wording, thanks!

**Pull request merged**